include/mbedtls/config_psa.h

```diff
@@ -675,19 +772,33 @@ extern "C" {
 #endif /* MBEDTLS_ECDSA_C */
 
 #if defined(MBEDTLS_ECP_C)
-#define MBEDTLS_PSA_BUILTIN_KEY_TYPE_ECC_KEY_PAIR 1
-#define PSA_WANT_KEY_TYPE_ECC_KEY_PAIR 1
+#define PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_USE 1
+#define PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_IMPORT 1
+#define PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_EXPORT 1
+#define PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_DERIVE 1
```

Comment:
Going by the general principle that `MBEDTLS_xxx` implies equivalent features in PSA, we shouldn't enable `DERIVE` here: deterministic derivation of EC keys doesn't exist in the legacy API.